guides/plugins/plugins/framework/flow/add-flow-builder-action.md

```diff
@@ -185,15 +183,29 @@ And don't forget to register your subscriber to the container at `<plugin root>/
 ```
 {% endcode %}
 
-Well done, you are successfully created your own custom action in Backend in PHP.
+Well done, you are successfully created your custom action in Backend in PHP.
 
 {% hint style="info" %}
   This functionality is available starting with Shopware 6.4.6.0
 {% endhint %}
 
 ## Add custom action in Administration
 
-- Define action name in `example-plugin.constant.js`
+After we have done with the PHP code, `action.create.tag` is received from the response of `/api/_info/flow-actions.json`. However, the custom action displays in the action list without label. These further steps in Administration will help you show the action label and add configuration for it.
+
+To see action list, we select a Trigger, for example [Example\Event] from the Trigger drop-down in the Flow tab. After that, we choose option `ACTION (THEN)`. A action component appears with action list.
+
+![Flow Builder trigger](../../../../../.gitbook/assets/flow-builder-trigger-drop.png)
+
+![Flow Builder action then](../../../../../.gitbook/assets/flow-builder-action-then.png)
+
+![Flow Builder trigger](../../../../../.gitbook/assets/flow-builder-action-no-label.png)
+
+### Step 1: Show action label in action list
+
+Firstly, we need to define an action name in `example-plugin.constant.js`. To be consistency with the custom action defined in PHP code, we also create the a constant called `CREATE_TAG` represent for `action.create.tag` which get from the response of `/api/_info/flow-actions.json`.
```

Comment:
"First, "

"To be consistent with the custom action defined in our PHP code, we also create a constant called `CREATE_TAG` to represent `action.create.tag` which get from the response of `/api/_info/flow-actions.json`."